src/lib/page-state/_core.scss

```diff
@@ -0,0 +1,84 @@
+@use './token-utils' as *;
+@use '../core/styles/typography';
+
+@forward './token-utils';
+
+@mixin host {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  container-type: inline-size;
+}
+
+@mixin host-hidden {
+  display: none;
+}
```

Comment:
Super minor, and I know I'm contradicting myself a little when I say this, but you can just revert this back to putting the individual style back in the `page-state.scss` file. I only say to do this because every custom element style has the same thing defined to mimic native HTML support, and it will never need to be a mixin that we intend to reuse. I don't mind in the end, but just wanted to share it with you as you go forward with other components so you know not worry about this special case selector.